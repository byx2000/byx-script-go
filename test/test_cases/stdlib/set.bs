import set

var set = Set()
set.add(1)
set.add(2)
set.add(3)
set.add(2)
var list = set.toList()
println(set.size())
println(set.contains(1))
println(set.contains(2))
println(set.contains(3))
println(set.contains(4))
println(set.remove(1))
println(set.remove(5))
println(set.size())
println(set.contains(1))
println(set.contains(2))
println(set.contains(3))
println(set.contains(4))
println(set.isEmpty())
set.remove(2)
set.remove(3)
println(set.isEmpty())

var s = Set()
for (var i = 0; i < 100; ++i) {
    s.add(i % 10)
}
println(s.size())
