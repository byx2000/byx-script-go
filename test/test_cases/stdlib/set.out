3
true
true
true
false
true
false
2
false
true
true
false
false
true
10